Makefile

```
@@ -48,3 +48,15 @@ install-grafana:
 		-v $(PWD)/collected-data/kpi_metrics.db:/var/lib/grafana/kpi_metrics.db:ro \
 		-e "GF_INSTALL_PLUGINS=frser-sqlite-datasource" \
 		grafana/grafana:$(GRAFANA_VERSION)
+
+
+# Install kpi-collector to user's Go bin directory
+install: build
+	mkdir -p $(HOME)/go/bin
+	cp $(BINARY_NAME) $(HOME)/go/bin/
+	echo "✓ Installed to $(HOME)/go/bin/$(BINARY_NAME)"
```

Since the <name> in cmd/<name> usually matches the binary name, you can also rely on "go install":
```suggestion
install:
	go install ./cmd/kpi-collector
	echo "✓ Installed to $(HOME)/go/bin/$(BINARY_NAME)"
```